### PR TITLE
Basic Docker compose to link up dependent services:

### PR DESCRIPTION
**Pull request by kimyoutora**
- Updated golang image to 1.6.0
- Compose configuration with postgres, redis, memcached

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
-FROM golang:1.5.1-onbuild
-EXPOSE 5000
\ No newline at end of file
+FROM golang:1.6.0-onbuild
+ADD . /app
```
Comment (kimyoutora) [inline Dockerfile:2]: puts the code in `/app` path in the image so we can easily find it in the compose config

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: '2' # omit this line iff version = 1
+services:
+  godo:
+    build: .
+    ports:
+      - "5000:5000"
+    volumes:
+      - .:/app
+    depends_on:
```
Comment (kimyoutora) [inline docker-compose.yml:9]: automatically links these up :)

File: README.md [hunk 3]
```diff
@@ -10,4 +10,14 @@ To run the application inside of docker on Mac run the following commands:
 
 `docker run -it --rm --name godo -P godo` Runs the container.
 
-To verify that everything is working `curl $(docker-machine ip default):$(docker port godo 5000 | awk -F: '{ print $2 }')/`.
\ No newline at end of file
+```bash
+# starts up the entire stack
+docker-compose up -d
```
Comment (kimyoutora) [inline README.md:15]: :boom:

**Comment by kimyoutora** [comment]
``` bash
 ~  workspace  …  github.com  kuriouslabs  godo  docker-compose  ✎  %  docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED              STATUS              PORTS                    NAMES
f0c08240e29c        godo_godo           "go-wrapper run"         39 seconds ago       Up 39 seconds       0.0.0.0:5000->5000/tcp   godo_godo_1
cc2faa9e078c        postgres            "/docker-entrypoint.s"   54 seconds ago       Up 54 seconds       5432/tcp                 godo_postgres_1
815dfa9e9c99        redis               "/entrypoint.sh redis"   About a minute ago   Up About a minute   6379/tcp                 godo_redis_1
767d0a424f91        memcached           "/entrypoint.sh memca"   About a minute ago   Up About a minute   11211/tcp                godo_memcached_1
```

**Comment by chaselatta** [comment]
I think this is missing things like creating a network and such so that these containers can talk to each other but that can come later. Looks good! :ship:

**Comment by kimyoutora** [comment]
the ports are automatically made availabe so godo can just query for localhost:11211 for memcached

**Pull request merged**